src/command.ts

```diff
@@ -507,13 +520,14 @@ export class Command<C extends Context = Context> implements MiddlewareObj<C> {
    */
   public toObject(
     languageCode: LanguageCode | "default" = "default",
-  ): BotCommand {
+  ): BotCommand & { noHandler?: boolean } {
```

Comment:
should we create our own BotCommand type? maybe AugmentedBotCommand or BotCommandX (taking inspiration from the hydration plug-in)

Reply:
we have something like that indeed with CommandElementals
https://github.com/grammyjs/commands/blob/4c9da16e8ef4014947a98b22da5cbabbe31c029a/src/types.ts#L37-L43
Maybe better to use that in the toObject function?

Reply:
Yeah I was just looking to be a lit bit more DRY

Reply:
I updated the interface to look like:
https://github.com/grammyjs/commands/blob/df13449f5df2596847fd1efc2143108aa3a307bc/src/types.ts#L37-L43
main difference between it an `BotCommand` is that it's property its called `name` instead of `command` for clarity but maybe that can be get rid of and we simply extend BotCommand

Reply:
> it would be less readable imo
because we'd have `command.command`?

Reply:
Yup

Reply:
Yup, but it's used so scarcely around the code that we could just make BotCommandX